docs/repositories-configure/configuring-code-patterns.md

```diff
@@ -1,12 +1,15 @@
 ---
-description: Configure the tools and code patterns that Codacy uses to analyze your repositories to create a coding standard adapted to your scenario.
+description: Configure the static analysis tools and code patterns that Codacy uses to analyze your repositories to ensure that the analysis results are adapted to your scenario.
 ---
 
 # Configuring code patterns
 
-By default, Codacy uses a subset of the code patterns that exist for each tool to analyze your repositories. These default code patterns result from community feedback or existing coding standards.
+By default, Codacy uses a subset of the supported static analysis tools and code patterns to analyze your repositories. These default settings result from community feedback or existing coding standards. However, you can adapt the default settings to your scenario by configuring the tools and code patterns that Codacy uses to analyze your repository.
 
-However, you can create a coding standard adapted to your scenario by configuring the tools and code patterns that Codacy uses to analyze each repository:
+!!! note
+    If your repository is following an [organization coding standard](../organizations/using-a-coding-standard.md), changes made to any tool or code pattern cause the repository to stop following the coding standard. In this case Codacy asks for your confirmation before accepting the changes.
```

Comment:
I added this note to make explicit that changing any tool or code pattern will require unlinking a repository from the coding standard it was following.